WireDomain/Sources/WireDomain/Synchronization/IncrementalSyncV2.swift

```diff
@@ -60,6 +63,9 @@ public struct IncrementalSyncV2: LiveSyncProtocol {
 
     public func perform() async throws -> IncrementalSync.Token {
         logger.debug("performing live sync", attributes: .syncAttributes(initialSync: false))
+
+        try await pullServerTimeSync.pull()
```

Comment:
question: so we only pull it once? The push channel will live longer so won't the time get out of date?

Reply:
What would happen though if the app is open for more than 30 seconds and receives a call event?  Wouldn't it be considered stale? Perhaps we can pull once at the start, then once we're up to date pull for every event.

Reply:
pulling for each event sounds not efficient, if that's really needed then better for backend to put the time within the event, no?

Reply:
actually that would have been the case also before since we received the servertime only when pulling pending events via /notifications not when opening the websocket

Reply:
So we never used up to date time stamps with the legacy push channel?

Reply:
@jullianm I checked with Android they're going to do as before so fetch before opening websocket